index.js

```diff
@@ -15,11 +17,65 @@ app.post("/", function(req, res, next) {
       var url = "https://calendar.google.com/calendar/embed?mode=AGENDA&amp;height=600&amp;wkst=1&amp;bgcolor=%23FFFFFF&amp;src=" + user + "%40goodeggs.com&amp;color=%2329527A&amp;ctz=America%2FLos_Angeles"
       res.status(200).send("Click to open <" + url + "|@" + user + "'s calendar>");
       break;
+
+    case "/helpscout":
+      var newConvo = req.body.text.split(" ");
+      var customer = newConvo[0];
+      var mailbox = newConvo[1];
+      var removeItems = newConvo.splice(0, 2);
+      var subject = newConvo.toString(",").replace(/,/g, " ");
```

Comment:
AFAIK `toString()` doesn't take an argument except in the special case of a number like `new Number(2).toString(2)` which will gave you the string representation in base 2 i.e. `10`.

So I think what's happening here is when you do `toString()` on an Array, you get back a comma separated representation e.g. `[1,2,3].toString()` = `'1,2,3'`, and then you're replacing the commas with spaces. This isn't great because if there were commas in the original message they'll be replaced as well. i think what you want is just `newConvo.join(' ')` which joins the elements array by whatever argument you pass in (so in this case `' '`) and returns a string.